pkgs/development/python-modules/brotlicffi/default.nix

```diff
@@ -4,12 +4,12 @@
 , pythonOlder
 , cffi
 , brotli
+, enum34
 }:
 
 buildPythonPackage rec {
   pname = "brotlicffi";
   version = "1.0.9.2";
-  disabled = pythonOlder "3.6";
```

Comment:
Why do we need Python 2 support? No package seems to need it. `requests` is the only consumer of `brotlicffi` and is using `brotli` for Python 2.

Python 2 is now EOL for 20 months.

Reply:
I'm not so sure that we care about python < 3.6 either. 3.5 is already EOL for over a year.